godotproject/Level0.tscn

```diff
@@ -33,6 +33,8 @@ margin = 10.0
 player_nodepath = NodePath("../Player")
 
 [node name="DeathPlane" parent="." instance=ExtResource( 4 )]
+level0_start_postion = Vector2( 121, 896 )
+level1_start_postion = Vector2( 121, 256 )
```

Comment:
In Level0.tscn, set start_position to the correct start position for level 0 (121, 896)
```suggestion
[node name="DeathPlane" parent="." instance=ExtResource( 4 )]
start_position = Vector2( 121, 896 )
```